pkg/rtc/participant.go

```diff
@@ -608,7 +608,7 @@ func (p *ParticipantImpl) SetMigrateInfo(
 	for _, t := range mediaTracks {
 		ti := t.GetTrack()
 
-		p.supervisor.AddPublication(livekit.TrackID(ti.Sid), ti.Type)
+		p.supervisor.AddPublication(livekit.TrackID(ti.Sid), ti.Type, nil)
```

Comment:
The ti.Type won't be used anymore since the prometheus call was moved to telemetry, can remove `ti.Type` here and update the supervisor.AddPublication signature to remove it.